README.md

```diff
@@ -66,10 +69,15 @@ OPML Import
 Selfoss supports importing OPML files. Find the OPML export in the old application, it is usually located somewhere in settings. Then visit the page https://yourselfossurl.com/opml and upload it there.
 
 
-APPS
+MOBILE APPS
 ----
 
-Two third party apps are available for Android: [Selfoss](https://play.google.com/store/apps/details?id=fr.ydelouis.selfoss) and [Reader For Selfoss](https://play.google.com/store/apps/details?id=apps.amine.bou.readerforselfoss).
+| App Url            | OS      | Project status | Compatibility* |
+|--------------------|---------|----------------|----------------|
+| [Reader For Selfoss](https://play.google.com/store/apps/details?id=apps.amine.bou.readerforselfoss) | Android | Active         | 2.18 and older |
+| [Selfoss](https://play.google.com/store/apps/details?id=fr.ydelouis.selfoss)            | Android | Inactive       | 2.18 and older |
```

Comment:
I would just remove the inactive app from the list.

Reply:
No, i use mobile app 'Selfoss' everyday, Ok the project is inactive but still compatible with Selfoss. I will use this this mobile app unless "Reader For Selfoss" add an "offline mode".

Reply:
Unfortunately, it does not work with 2.19 so we will have to remove it anyway. Hopefully, by the time it is released, Reader for Selfoss will support offline mode: https://github.com/aminecmi/ReaderforSelfoss/issues/179

Reply:
@Binnette There is now an offline mode.